code/game/objects/effects/decals/cleanable/humans.dm

```diff
@@ -6,6 +6,7 @@
 	random_icon_states = list("floor1", "floor2", "floor3", "floor4", "floor5", "floor6", "floor7")
 	blood_state = BLOOD_STATE_HUMAN
 	bloodiness = BLOOD_AMOUNT_PER_DECAL
+	plane = -1
```

Comment:
magical number bad

Reply:
It just deletes itself when it collides with a wall if this doesn't get assigned

Reply:
Nevermind, it doesn't delete itself, it only makes the blood appear beneath the wall, which isn't right.